### PR TITLE
Setup fastapi htmx

### LINKED ISSUE #2: Implement MVP for a page with a list of project tasks grouped per status
Create a page with a list of project tasks. The page contains a bar and three columns that are related to the status of a task. See sketch below.

A task is presented as "a card"and when the task is assigned, the user's avatar is visible on the task.

If the task has no status, it will be presented in the "todo" bar. A user can drag & drop a task between the "todo" bar and status columns. A drag & drop event will update the status of the task. 

A few assumptions:
- user is logged in
- user is linked to:
  - an organisation
  - a project within the organisation
  - a role within the project

Image

When a task is moved to done, details (such as a project description) are git committed to a system card.

Depends on https://github.com/orgs/MinBZK/projects/7/views/6?pane=issue&itemId=60092144

# Requirements

- There should be CI/CD
- The MVP should be deployed on k8s
- There should be test coverage
- There should be an end-to-end test

### PR DESCRIPTION
**Pull request by berrydenhartog**
# Description

Here i implemented the basic fastapi setup. this includes:
* Basic logging
* Simple Readme 
* Custom exception handelers
* Custom default response class
* Vscode setup imrpovements
* 1 jinja template (to be created more in a new PR)
* Database migrations (with alembic)
* SQLMode (1 example model)
* Tests
* CI/CD improvements
* Dependency updates
* docker-compose with postgres

Resolves #2 

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x]  I have tested the changes locally and verified that they work as expected.
- [x]  I have followed the project's coding conventions and style guidelines.
- [x]  I have rebased my branch onto the latest commit of the main branch.
- [x]  I have squashed or reorganized my commits into logical units.
- [x]  I have read, understood and agree to the [Developer Certificate of Origin](../blob/main/DCO.md), which this project utilizes.

### CODE REVIEW

**Comment by anneschuth** [comment]
Glanced over most of this and it looks great!

We're not running end-to-end test (including, for instance, docker build steps)?

**Comment by berrydenhartog** [comment]
> Glanced over most of this and it looks great!
> 
> We're not running end-to-end test (including, for instance, docker build steps)?

Not yet. That will be added in the last PR (when we have the frontend done)

File: BUILD.md [hunk 1]
```diff
@@ -20,7 +20,23 @@ poetry install
 When poetry is done installing all dependencies you can start using the tool.
 
 ```shell
-poetry run python -m tad
+poetry run python -m uvicorn tad.main:app
```
Comment (uittenbroekrobbert) [inline BUILD.md:23]: This is the 'old' startup, can you update it?

Comment (berrydenhartog) [reply BUILD.md]: Done

**Comment by anneschuth** [comment]
Right, I saw the build but didn't see it covered in tests. Wouldn't we want to be able to run the whole test suite (including builds) locally as well? 

Fine if all this happens in a follow up PR

**Comment by berrydenhartog** [comment]
One part i 

> Right, I saw the build but didn't see it covered in tests. Wouldn't we want to be able to run the whole test suite (including builds) locally as well?
> 
> Fine if all this happens in a follow up PR

i am not sure if i will build it the way you want it. we will see when it is done...

**Comment by uittenbroekrobbert** [comment]
Compliments on an extremely well setup project @berrydenhartog ! Happy to approve this and continue with more awesome things!

**Review by uittenbroekrobbert — APPROVED** [review]

**Pull request merged**